node/api/server.go

```diff
@@ -64,6 +65,13 @@ func PodHandler(p PodHandlerConfig, debug bool) http.Handler {
 			WithExecStreamIdleTimeout(p.StreamIdleTimeout),
 		),
 	).Methods("POST", "GET")
+
+	if p.GetStatsSummary != nil {
```

Comment:
Just a nit, `HandlePotStatsSummary` will return a `NotImplemented` handler if this is nil anyway.